### PR TITLE
import SuperstaQJob/Backend/Provider in superstaq/__init__.py

### LINKED ISSUE #137: import SuperstaQJob/Backend/Provider in superstaq/__init__.py
it is cumbersome for us to type 
`qss.superstaq_provider.SuperstaQProvider`

### PR DESCRIPTION
**Pull request by singular-value**
Closes #137

### CODE REVIEW

**Comment by richrines1** [comment]
nice, this one has been bothering me.

fwiw there are still a few instances of `qiskit_superstaq.superstaq_provider.SuperstaQProvider` in daily_integration_test.py that could be updated

**Comment by singular-value** [comment]
> fwiw there are still a few instances of qiskit_superstaq.superstaq_provider.SuperstaQProvider in daily_integration_test.py that could be updated
Done

File: qiskit_superstaq/daily_integration_test.py [hunk 1]
```diff
@@ -6,19 +6,19 @@
 import qiskit
 from applications_superstaq import SuperstaQException
 
-import qiskit_superstaq
+import qiskit_superstaq as qss
```
Comment (richrines1) [inline qiskit_superstaq/daily_integration_test.py:9]: do we want to change this? #26

File: qiskit_superstaq/superstaq_provider.py [hunk 2]
```diff
@@ -36,7 +36,7 @@ class SuperstaQProvider(
 
         import qiskit_superstaq as qss
 
-        ss_provider = qss.superstaq_provider.SuperstaQProvider('MY_TOKEN')
+        ss_provider = qss.SuperstaQProvider('MY_TOKEN')
```
Comment (richrines1) [inline qiskit_superstaq/superstaq_provider.py:39]:
```suggestion
        ss_provider = qss.SuperstaQProvider(api_key='MY_TOKEN')
```
(also do you remember why we didn't make api_key the first argument?)

Comment (singular-value) [reply qiskit_superstaq/superstaq_provider.py:39]: Done

**Comment by singular-value** [comment]
@vtomole @richrines1 Ready for review again

File: qiskit_superstaq/superstaq_provider.py (hunk 2 above)
Comment (richrines1) [reply qiskit_superstaq/superstaq_provider.py:39]: we should do the same thing for `css.Service`

**Review by richrines1 — APPROVED** [review]
lgtm!

File: qiskit_superstaq/superstaq_provider.py (hunk 2 above)
Comment (singular-value) [reply qiskit_superstaq/superstaq_provider.py:39]: ^ https://github.com/SupertechLabs/cirq-superstaq/issues/261

**Pull request merged**